### PR TITLE
Update to pycapnp v0.5.9 and to conda-smithy 2.2.2

### PR DESCRIPTION
**Pull request by lannybroo**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: pycapnp-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pycapnp/pycapnp-{{ version }}.tar.gz
-  md5: d47396d6724593afcbb728b87ac6de07
+  md5: 4180c74f4a9adfcd8a5a605843399c83
```
Comment (jakirkham) [inline recipe/meta.yaml]: Probably should switch to `sha256` from [pypi.org]( http://pypi.org/ ).

**Comment by jakirkham** [comment]
Glad to see you were able to pick up `conda-smithy` ok. If you have any questions or concerns about your experience, feel free to raise them over at the [`conda-smithy` repo]( http://github.com/conda-forge/conda-smithy ).

**Comment by frol** [comment]
It seems that the `build.sh` should be updated with

```
export MACOSX_DEPLOYMENT_TARGET=10.9
```

File: recipe/meta.yaml (hunk 1 above)
Comment (lannybroo) [inline recipe/meta.yaml]: Agreed. Also realized the latest pycapnp is 0.5.10, so I'll update to that as well.

**Pull request closed without merging**